sound/soc/sof/sof-client-dma-trace.c

```diff
@@ -0,0 +1,673 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
```

Comment:
Is this file a preliminary option, so far just copying code, and the final version will reuse the code or remove it from its current location?

Reply:
I would prefer GPL-2.0-only to be honest. @lgirdwood ?

Reply:
@ujfalusi sorry, I didn't mean that specific line, I meant the whole file. GH doesn't let you comment to files or to whole commits

Reply:
It is mostly a copy, adopted for cdev and the auxbus registration as a separated, standalone device.
The final version will remove the old code and all traces of the 'trace' from sdev and core SOF code.

Reply:
right, thanks, if possible - would be good to do that in a review-friendly way. I.e. with some renames or at least with preserving order of functions in the new copy, for easier diff-ing?